task-management/src/main/webapp/vue-app/tasks-management/components/Project/ProjectEventFormAssigneeAndParticipatorItem.vue

```diff
@@ -38,8 +43,17 @@ export default {
     },
     displayName() {
       const profile = this.attendee && (this.attendee.profile || this.attendee.space);
-      return profile && (profile.displayName || profile.fullname || profile.fullName);
+      const fullName = profile && (profile.displayName || profile.fullname || profile.fullName);
+      return this.isExternal && fullName.indexOf(this.$t('label.external')) < 0 ? fullName.concat(' (').concat(this.$t('label.external')).concat(')') : fullName;
```

Comment:
Why do you need this test: fullName.indexOf(this.$t('label.external')) < 0

Reply:
This test to assure that we have a unique external tag when we display users after selections